students/woogyeong/westagram/users/views.py

```diff
@@ -33,4 +33,24 @@ def post(self, request):
             return JsonResponse({'message': "CREATED"}, status=201)
         except KeyError:
             return JsonResponse({'message': 'KEY_ERROR'}, status=400)
-        
\ No newline at end of file
+        
+class LogInView(View):
+    def post(self, request):
+        data     = json.loads(request.body)
+        email    = data['email']
+        password = data['password']
+
+        try:
+            user    = User.objects.get(email=email)
```

Comment:
user가 존재하는지 여부 판단 이전에 get을 통해 가져오게 되면 DB에 user가 없을 때 에러가 발생하게 됩니다.
따라서 전체적으로 코드가 다음과 같이 흘러가면 될 것 같습니다.

JWT를 적용하면 SUCCESS 대신에 토큰을 전달할 예정이지만, 지금 미션에서는 success만으로 충분합니다.
```suggestion
           if not User.objects.filter(email=email, password=password).exists():
                return JsonResponse({'message': 'INVALID_USER'}, status=401)
           
           return JsonResponse({'message': 'SUCCESS'}, status=200)
```